iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ProportionalScalable.java

```diff
@@ -69,8 +75,137 @@ void setIterationPercentage(double iterationPercentage) {
         }
     }
 
+    public double scaleOnLoads(Network network,
+                               Reporter subReporter,
+                               VariationParameters variationParameters,
+                               Collection<Load> loads) {
+        // Initialisation of the ProportionalScalable
+        ProportionalScalable proportionalScalable = new ProportionalScalable();
```

Comment:
At the beginning I wanted to have two constructors (one having Collection<Loads> in parameter, the other one Collection<Generators>), but it raised warnings/errors since having two constructors with the same parameters (the only difference was between the <>) isn't a good way to code.
The solution we found was then to use two public methods (scaleOnLoads and scaleOnGenerators) that will create the ProportionalScalable. I agree it's a bit strange so if you have a better idea, I'll be glad to hear it!

Reply:
So what you want here is to create a utility static method

Reply:
should I pull the two methods out of this class and into some other utility class or only adding the "static" is enough ?